library/Zend/Code/Scanner/ClassScanner.php

```diff
@@ -480,6 +505,142 @@ public function hasProperty($name)
         return false;
     }
 
+
+    public function getTraits()
+    {
+        if(empty($this->traits)) {
+            //get list of traitNames
+            $traitNames = $this->getTraitNames();
+            foreach($traitNames as $traitName) {
+                /*
+                 * @todo test if this works when trait is included or required instead of autoloaded?
+                 */
+                $fileName = str_replace("\\", "/", $traitName) .".php";
```

Comment:
Hmmm... this may not be the best approach; the point of the `Scanner` subcomponent is to allow scanning without reflection. I'm going to leave with my changes for now; we can see if there's a better way to accomplish this later.